plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -3037,6 +3048,26 @@ def get_authz_resource_by_name(self, name, client_id, realm):
         except Exception:
             return False
 
+    def remove_authz_permission(self, id, client_id, realm):
+        """Remove an authorization resource for a Keycloak client"""
+        url = URL_AUTHZ_RESOURCE.format(url=self.baseurl, id=id, client_id=client_id, realm=realm)
+
+        try:
+            return open_url(url, method='DELETE', http_agent=self.http_agent, headers=self.restheaders, timeout=self.connection_timeout,
+                            validate_certs=self.validate_certs)
+        except Exception as e:
+            self.fail_open_url(e, msg='Could not delete permission %s for client %s in realm %s: %s' % (id, client_id, realm, str(e)))
```

Comment:
Nitpick: I found a few places which refer to the resource as a permission or a scope.
```suggestion
            self.fail_open_url(e, msg='Could not delete resource %s for client %s in realm %s: %s' % (id, client_id, realm, str(e)))
```